deepspeed/runtime/zero/stage2.py

```diff
@@ -1532,6 +1538,9 @@ def has_overflow_partitioned_grads_serial(self):
 
     def has_overflow(self, partition_gradients=True):
         if partition_gradients:
+            if self.overlap_comm:
```

Comment:
if self.overlap_comm and self.cpu_offload

Reply:
It should be fixed now.